src/NoteDataUtil.cpp

```diff
@@ -1855,6 +1855,144 @@ static void GetTrackMapping( StepsType st, NoteDataUtil::TrackMapping tt, int Nu
 	}
 }
 
+static void HyperShuffleNotes( NoteData &inout, int iStartIndex, int iEndIndex)
+{
+	int iNumTracks = inout.GetNumTracks();
+
+	// Stores the list of tracks that can have notes placed into them.
+	// This will be shuffled so that notes can be placed in different tracks.
+	vector<int> viTargetTracks;
```

Comment:
Both of these vectors aren't used until the `FOREACH_NONEMPTY_ROW_ALL_TRACKS_RANGE` loop below. Move it closer to that.